app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt

```diff
@@ -44,6 +44,8 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 import timber.log.Timber
 import org.fossasia.openevent.general.utils.Utils.setToolbar
 
+const val FILTER_EVENTS: Int = 1
```

Comment:
Not needed

Reply:
so what parameter to send on chip click?

Reply:
Boolean